Gang Mode Update 4

Ikarrus opened the pull request:
**Major Changes**
- Increased win condition to require 66% control
- Added a switchblade. A relatively cheap and decently robust melee weapon for gangsters.
- Limited gang spraycan use to 15
- Promotions now start off cheap at 20 influence but get more expensive the more people you promote. As a consequence, you can only promote up to 3 lieutenants.

**Minor Changes**
- Increased pen cooldown to (15*Gang size)+45 seconds
- Tagging a territory now tells you what territory you actually tagged!
- Added an option to silence the verbose territorial updates you get from gangtools
- Capped how much influence you can get every income at 50
- Updated Centcom report on gangs
- Spraycans always show their inhands now, instead of only when uncapped
- Antag panel now shows how much influence each gang has
- Cleaned up gang code in the traitor panel
- Added logging for gangtool purchases
- Promotion cost now counts how many times you bought that item instead of how many bosses you have, so you can't stockpile on cheap gangtools before promoting anyone
- Fixed non-gangers getting gang messages when trying to spray on an invalid territory with a gang spraycan

Ikarrus committed:
Gang Mode Update 4

**Major Changes**
- Increased win condition to require 66%
- Added a switchblade. A relatively cheap and decently robust melee weapon for gangsters.
- Capped how much influence you can get every income at 50
- Limited gang spraycan use to 15
- Pistol cost increased to 30 Influence
- Promotion cost now scales to how many bosses you already have. The more bosses, the more expensive it will be. It starts off cheap at 20 influence now.

**Minor Changes**
- Tagging a territory now tells you what territory you actually tagged!
- Added an option to silence the verbose territorial updates you get from gangtools
- Capped how much influence you can get every income at 50
- Updated Centcom report on gangs
- Spraycans always show their inhands now, instead of only when uncapped
- Antag panel now shows how much influence each gang has
- Cleaned up gang code in the traitor panel
- Fixed non-gangers getting gang messages when trying to spray on an invalid territory

Ikarrus committed:
Added a changelog and additional tweaks

Lumipharon commented:
Fuck yeah, switchblades were fun as shit on NT.

Could they be constructable also? If nothing else but for a bit of anti meta. NT had them need metal and plasteel, so they weren't super easy to make (although eva tended to get raided a bit).

Alek2ander commented:
Hacked autolathe?

Ikarrus committed:
- Increased pen cost to 60, mostly so you can't be buying one every 5 minutes late round
- Increased pen cooldown to (15*Gang size)+45 seconds
- Promotion cost now counts how many times you bought that item instead of how many bosses you have, so you can't stockpile on cheap gangtools before promoting anyone

Ikarrus committed:
Resolve Conflicts:
	icons/mob/inhands/items_lefthand.dmi
	icons/mob/inhands/items_righthand.dmi
	icons/obj/weapons.dmi

tedward1337 commented:
Fuck yeah, switchblades

ike709 commented:
make switchblades constructable. it brings us one step closer to NT's glorious gun cargo.

:+1:

Ikarrus committed:
- Switchblade changes hitsounds now if it's not extended.
- Reverted the boss icon back to use a B. I changed it for a different idea I had but I never went through with it but I forgot to change the icon back
- Price tweaks

Ikarrus committed:
Fixed switchblade hitsound not starting in its retracted state

Fox-McCloud commented:
I don't think a concealable weapon that can do, literally, 20 damage per stab-----would be a good thing to be constructable by crewmembers.

The best weapon available to crew is the fire-axe---which does 24 and requires it to be *wielded*.

Cookiesndip commented:
> Increased pen cooldown to (15*Gang size)+45 seconds

What was it previously, for scale? I've found that very little of the station actually winds up in a gang even with zero implanting going on based on rounds yesterday. Reducing the availability of implants is a whole new can of worms and would be entailing entirely stripping rev, so let's not even consider that. Optimally, the two gangs together should have the numbers and the strength to fight on par with security in the midgame (by then everything is tagged, borders drawn, etc.) if they become an obstruction/forceimplant folks.

Constraining the paint to 15/per purchase without lowering the price in proportion, will excaberate this further, as paint will accelerate victory AND provide additional/protect your income, whereas, say, a pistol is almost completely useless at such a high price, being able to kill/protect you from 2~ people, when a slip/stunbaton and saw or welder fluid + lighter would be just as effective, or one of the billion other combat options.

Optimally, taggers should be the most expensive items, by far, whereas consumables (syndicate pistols with limited ammo and switchblades esp.) should be markedly cheaper. There isn't much you can do with a pistol with no extra ammo (and a slip + saw is just as effective), and, unless the switchblade is markedly better than normal melee units, you're better off not buying it and using a station-available item instead. Perhaps, if switchblades do not have an onhand sprite and have saw-level damage, that might be enough to justify purchase and opportunity cost.

phil235 commented on code/game/objects/items/weapons/weaponry.dm:
```diff
@@ -158,4 +158,40 @@ obj/item/weapon/wirerod/attackby(var/obj/item/I, mob/user as mob, params)
+	icon_state = "switchblade"
+	desc = "A sharp, concealable, spring-loaded knife."
+	flags = CONDUCT
+	force = 20
```
You weren't joking when you said these were robust!

Miauw62 replied:
20 is really robust. make it like 16 or something, it'd still be pretty robust.

Ikarrus replied:
16 is just barely stronger than a welding tool, which at that point makes the item utterly useless to spend influence on.

Miauw62 replied:
as somebody else noted, 20 is only 4 less than a fire axe and those have to be wielded.

Ikarrus replied:
And I'm fine with that! 20 is what I would consider to be the minimally acceptable force for the weapon to have any value at all.

Keep in mind that in order to get one you have to spend *valuable* influence that isn't just to buy toys, but also to keep your gang going with spraycans.

Ikarrus committed:
Pistols -> 25 Cost

Fox-McCloud commented:
For the record, I"m not suggesting that gang version be nerfed, just that the gang version of 20 brute NOT be constructable/obtainable by crew and that something that you can conceal in your pocket that does that much damage (and is for non-antags) is OP.
